pkgs/tools/archivers/wimlib/default.nix

```diff
@@ -8,15 +8,15 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "1.13.2";
+  version = "1.13.3";
   pname = "wimlib";
 
   nativeBuildInputs = [ pkg-config makeWrapper ];
   buildInputs = [ openssl fuse libxml2 ntfs3g ];
 
   src = fetchurl {
     url = "https://wimlib.net/downloads/${pname}-${version}.tar.gz";
-    sha256 = "0id9ym3hzij4kpdrk0sz3ijxp5r0z1md5jch83pml9hdy1zbx5bj";
+    sha256 = "sha256-igdB0H2TFHNbBAzqYWj22vGsHHLTUNcD8oaxGBNd+n4=";
```

Comment:
Please don't use the the new style of hashes just yet. The attribute is called `sha256` thus we should pass a base32 sha256 sum instead of the SRI style hash. Does this even work in our min version Nix?

Reply:
This is done all over the place. https://search.nix.gsc.io/?q=sha256%20%3D%20%22sha256-&i=nope&files=&repos= counts over 1600 ocurences.